LoopKitUI/Extensions/Environment+Authenticate.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Environment+Authenticate.swift
+//  LoopKitUI
+//
+//  Created by Rick Pasetto on 8/3/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import LocalAuthentication
+import SwiftUI
+
+public typealias Completion<Value> = (Result<Value, Error>) -> Void
```

Comment:
[nit] I don't think this typealias is necessary

Reply:
Ok.  I think someday we may want it ;)